src/stability_sdk/client.py

```diff
@@ -308,7 +345,7 @@ def generate(
                 seed=seed,
                 steps=steps,
                 samples=samples,
-                parameters=parameters,
+                parameters=[generation.StepParameter(**step_parameters)],
```

Comment:
As commented earlier, the `step_parameters` a `[generation.StepParameter()]`.

Reply:
see response to earlier comment